indent.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Install emacs: 
+#	  sudo apt-get install emacs
+#
+# Get the indentation emacs script as follows: 
+#  	wget https://raw.github.com/Glavin001/atom-beautify/master/src/beautifiers/fortran-beautifier/emacs-fortran-formating-script.lisp
+# 
+# Change to downcase: 
+#   sed 's/-upcase-/-downcase-/g' emacs-fortran-formating-script.lisp > emacs-fortran-formating-script-down.lisp
+#
+# To use it just call this script from the main LATTE folder. 
+# 	./tools/indent.sh
+
+PROGRESS_PATH=$HOME/qmd-progress/
```

Comment:
This only works if you clone the repository into `${HOME}`. Try something like

`PROGRESS_PATH=$(dirname $(readlink -f $0))`